### ISSUE TITLE
Bring Dynamic Trees to the next level

### ISSUE THREAD

**Issue by lllMohandarlll** [ferreusveritas/Dynamic-Trees#387]
So, basically DynamicTrees aim to make trees better; more... dynamic in fact...
On 1.12.2 you implemented a very cool animation where tree litterally fall after being cut down on, then once the animation is done the tree just drops the materials (logs etz).
An interesting prosecute of this behavior, or let's say, litterally the next step, would be that after the animation, the cut tree stays on ground, and the player have the option to cut it in 2/3/4 pieces (based on the tree growth/lenght), and use other mod/system for transportation, or just further cut the tree pieces to get the drop (vanilla logs).
This concept it's inspired (and pretty the same) of Medieval Engineers
here some screenshot
![image](https://user-images.githubusercontent.com/11860704/71558172-88da9c80-2a50-11ea-9787-d8165517396f.png)
![image](https://user-images.githubusercontent.com/11860704/71558176-942dc800-2a50-11ea-9af1-dba83660105a.png)

Also, this new feature would be pretty interesting for eventual transportation mods (or future mods/modpacks) to create an interesting system of wood gathering on large scale
![image](https://user-images.githubusercontent.com/11860704/71558213-eb339d00-2a50-11ea-9294-26d64c097cdb.png)

**Comment by kierajreed** [comment]
This is a really cool idea, but unfortunately out of scope of this project. Sorry.

**Comment by ghost** [comment]
Agreed that it's a cool idea.  Aside from the truck/train/transport piece(which is definitely out of scope) it's something I've considered before.  The problem is that if you go to cut up the logs from the fallen tree you will more often than not end up with incomplete portions of a log.  This is an inventory problem with having an unreasonable collection of tree pieces of various dimensions and wood types that don't stack.  I found that any way around this would be extremely inconvenient or needlessly complex.

**Comment by lllMohandarlll** [comment]
> The problem is that if you go to cut up the logs from the fallen tree you will more often than not end up with incomplete portions of a log.

I may have an idea for this, I don't know if it's viable anyhow..
One possible workaround would be to insert some custom items (custom wood pieces) for incomplete logs and branches, every stage of the growing tree drop a different quantity of these custom items (called, idk, wood maybe)... i've noticed that on nbt there are around 7 stages of growth... 
so based upon the stage the piece wll drop a certain quantity of these custom items;
then these chunks of wood could then be converted into vanilla logs or mod's trees logs simply using the crafting gui of the player (4 custom items for 1 vanilla log for example)

**Comment by kierajreed** [comment]
Too complicated and inelegant. Nah.

**Comment by lllMohandarlll** [comment]
Another simpler solution would be, stages 4-7 drops normal logs, (1 each)
stages 1-3 drops branches/or just vanilla sticks (1 each)

(can't think about something simpler than this xD)
